sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -633,9 +685,14 @@ def to_runner_api(self, context):
 
 class TriggerContext(object):
 
-  def __init__(self, outer, window):
+  def __init__(self, outer, window, clock):
     self._outer = outer
     self._window = window
+    self._clock = clock
+
+  @property
```

Comment:
<!--thread_id:cc_163034547_t; commit:5ddb81e7534625fa7f44e5ab7dbada5eb605bf3b; resolved:0-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Let's not make this `@property` to make it clearer that this is actually retrieving the time on each call.

<!--section:body-->
OK, I am changing it (I guess that is how it is done in the project), although I don’t quite see why having it as a property it could be assumed not to change (as much as any other attribute in python would).